packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -393,6 +393,11 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 				font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
 			}
 
+			.tox .tox-dialog-wrap__backdrop {
+				background-color: ${theme.backgroundColor} !important;
+				opacity:0.7
```

Comment:
What's the default opacity?

Reply:
previously there was no default opacity . The css looked something like this:
```
.tox .tox-dialog-wrap__backdrop {
    background-color: rgba(255,255,255,.75);
    bottom: 0;
    left: 0;
    position: absolute;
    right: 0;
    top: 0;
    z-index: 1;
}
```
Due to `rgba` it was having some kind of semi-transparent white colour .